### ISSUE TITLE
v. 2.3.4 Arbalest item config doesn't working

### ISSUE THREAD

**Issue by Ksyphos** [Wacky-Mole/WackysDatabase#16]
Edits in "Item_CrossbowArbalest.yml" wont make effect in game.

**Comment by Wacky-Mole** [comment]
Yep, that's pretty weird, it does change the values after reloading 4 times. I don't know why

**Comment by Ksyphos** [comment]
Weird indeed. Had to use Custom weapon stats(despite it limited functionality) exclusively for arbalest.

**Comment by Wacky-Mole** [comment]
Cloning the Crossbow works fine. Clone and disable original item/recipe

**Comment by Runolv** [comment]
looks like same thing for  BattleaxeBlackmetal

**Comment by Wacky-Mole** [comment]
Thank you all for your help. This issue is finally fixed in 2.4.79
